mkt/search/utils.py

```diff
@@ -9,3 +15,61 @@ def execute(self):
             results = super(Search, self).execute()
             statsd.timing('search.took', results.took)
             return results
+
+
+def _property_value_by_region(obj, region=None, property=None):
+    if obj.is_dummy_content_for_qa():
+        # Apps and Websites set up by QA for testing should never be considered
+        # popular or trending.
+        return 0
+
+    if region and not region.adolescent:
+        by_region = region.id
+    else:
+        by_region = 0
+
+    try:
+        return getattr(obj, property).get(region=by_region).value
```

Comment:
The related manager - see `get_popularity()`/`get_trending()` methods below. I can add a comment to make that clear.